docs/rdd-programming-guide.md

```diff
@@ -1256,7 +1256,7 @@ storage levels is:
 
 **Note:** *In Python, stored objects will always be serialized with the [Pickle](https://docs.python.org/2/library/pickle.html) library,
 so it does not matter whether you choose a serialized level. The available storage levels in Python include `MEMORY_ONLY`, `MEMORY_ONLY_2`,
-`MEMORY_AND_DISK`, `MEMORY_AND_DISK_2`, `DISK_ONLY`, and `DISK_ONLY_2`.*
+`MEMORY_AND_DISK`, `MEMORY_AND_DISK_2`, `DISK_ONLY`, `DISK_ONLY_2`, and `DISK_ONLY_3`.*
```

Comment:
it looks like we need to update the table above as well?
it might be nice to say what happens if you specify a level > 1 but you don't have that many executors.

Reply:
Let me rephrase the request.
1. Adding both DISK_ONLY_2 and DISK_ONLY_3 to the above table.
2. Adding a description about the corner case for `MEMORY_ONLY_2`, `MEMORY_AND_DISK_2`, `DISK_ONLY_2`, `DISK_ONLY_3`

Is there something more I can do, @tgravescs ?

Reply:
thats it.